src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -75,7 +75,7 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 	nodeBinary := fmt.Sprintf("%s/bin/node", consts.NodeInstallationDir)
 	enableDynamicInstall := common.GetBooleanEnvironmentVariable(consts.EnableDynamicInstallKey)
 	if enableDynamicInstall && !common.PathExists(nodeBinary) {
-		scriptBuilder.WriteString(fmt.Sprintf("oryx setupEnv -appPath %s\n", gen.SourcePath))
+		scriptBuilder.WriteString(fmt.Sprintf("oryx prep --src %s\n", appPath))
```

Comment:
why? i would not change this  for now

Reply:
`oryx setupEnv` fails when invoked from the generated `/tmp/run.sh` in the latest build image. `oryx prep` works. Will this be a breaking change for the run image? If not may you elaborate on why we should hold off for now?

Reply:
do we know why setupenv fails when its called from run.sh?

oryx prep is prepare build environment , oryx setupEnv is for preparing runtime environment. We should not mix the usage of these two commands, ideally we should use only one command from now on as we are consolidating both build and runtime environment in one image, but I am not expecting that consolidation of prep and setup command in this PR. That can be done later. But I would be curious to know why setupEnv command is failing before I use prep command

Reply:
@arroyc I see. `oryx setupEnv` since it is not implemented in the `latest` build image, which make sense now. I've reverted back to `oryx setupEnv` so we could consolidate later